lms/templates/courseware/courseware.html

```diff
@@ -73,7 +76,7 @@
   <script type="text/javascript" src="${static.url('js/vendor/codemirror-compressed.js')}"></script>
 
   <%static:js group='courseware'/>
-  <%include file="../discussion/_js_body_dependencies.html" />
+  <%include file="/mathjax_include.html" args="disable_fast_preview=disable_fast_preview"/>
```

Comment:
Thanks for clarifying. The code is also now much clearer in that it makes the MathJax dependency explicit, rather than having it hidden in a discussion file. I guess I have to wait until all xmodules are converted to XBlocks before we can stop loading MathJax on every page.